macros/edr/tests/test_utils/get_anomaly_query.sql

```diff
@@ -50,3 +50,13 @@
     {%- endset -%}
     {{- return(anomaly_query) -}}
 {% endmacro %}
+
+{%- macro is_score_anomalous_condition(sensitivity) -%}
+    {%- set spikes_only_metrics = ['freshness', 'event_freshness'] -%}
+
+    case when metric_name IN {{ elementary.strings_list_to_tuple(spikes_only_metrics) }} then
+            anomaly_score > {{ sensitivity }}
+         else
+            abs(anomaly_score) > {{ sensitivity }}
+         end
+{%- endmacro -%}
```

Comment:
For freshness, a lower metric value is a good thing.
So we only want to detect spikes (positive anomaly score beyond threshold)